base/setup/usetup/usetup.c

```diff
@@ -526,257 +526,11 @@ GetNTOSInstallationName(
     }
 }
 
+//Launage Page.
+#include "headers/usetupLaunagePage.h"
```

Comment:
Launage, and also why is this capitalized?
The rest of the headers are not capitalized.

Reply:
I do not think it matters. I also cannot find any other headers in that directory that would benefit from camelcase.